src/components/30-organisms/modal/ui.test.js

```diff
@@ -0,0 +1,19 @@
+import { Selector } from 'testcafe';
```

Comment:
This file needs to be removed and replaced by one called `pw.ui.test.js`. In there you will have to add some tests that validate the functionality of the modal.

Reply:
You can add this to the issue for "create-amo" script, too.